lib/Init.vala

```diff
@@ -81,6 +84,9 @@ namespace Granite {
             if (base_provider == null) {
                 base_provider = new Gtk.CssProvider ();
                 base_provider.load_from_resource ("/io/elementary/granite/Granite.css");
+                #if INCLUDE_GTK
+                    base_provider.load_from_resource ("/io/elementary/granite/Gtk.css");
```

Comment:
According to docs the `load` functions clear any previously loaded resources, so you'd need to add a third and fourth provider for GTK stylesheets